_posts/2020-05-01-newsletter.md

```diff
@@ -0,0 +1,163 @@
+
+- Neovim now always waits for a UI when usied with --embed, unless --headless is also supplied (**@bfredl, [#9024](https://github.com/neovim/neovim/pull/9024)).
+
+- Neovim's libuv loop has been exposed to lua as `vim.loop` (**@zhaozg**, **@andreypopp**, [#10123](https://github.com/neovim/neovim/pull/10123)).
+
+### The near future: Upcoming
+
+
+Around Neovim
+-------------
+
+
+## Related projects
+
+The wiki page of [related
+projects](https://github.com/neovim/neovim/wiki/Related-projects) is an evergrowing list of projects that somehow make use of Neovim's extended capabilities. Here's a gist of additions since the last newsletter: 
+
+### Plugins
+
+The following plugins have found their way into our [wiki](https://github.com/neovim/neovim/wiki):
+
+- [coc.nvim](https://github.com/neoclide/coc.nvim): Language Server Protocol client that aims to replicate VSCode's functionnalities
+- [markdown-preview.nvim](https://github.com/iamcco/markdown-preview.nvim): Provides preview of your markdown files in your browser, with synchronized scrolling.
+- [nvim-gdb](https://github.com/sakhnik/nvim-gdb): Neovim thin wrapper for GDB, LLDB and PDB
+- [nero.nvim](https://github.com/Vigemus/nero.nvim): A REPL for Neovim
+- [neogdb.vim](https://github.com/huawenyu/neogdb.vim): Vim GDB front-end for neovim
+- [nvim-luadev](https://github.com/bfredl/nvim-luadev): REPL for developing lua plugins
+- [Conjure](https://github.com/Olical/conjure): Clojure and ClojureScript tooling for Neovim over a socket
+
+
+### API Clients
+
+Plenty of new API clients:
+
+- [neovim-java](https://github.com/esensar/neovim-java)
+- [viniarck/nvimhost-d](https://github.com/viniarck/nvimhost-d)
+- [viniarck/nvimhost-scala](https://github.com/viniarck/nvimhost-scala)
+
+### GUIs
+
+Maybe 2018/2019 weren't the years of the Linux desktop, but they sure were the years of the Neovim GUIs:
+
+- [veonim](https://github.com/veonim/veonim): a really cool electron-based GUI that aims to replicate the VSCode experience.
+- [DinVim Vim for Mac](http://dinvim.com): A "safe and secure" Neovim GUI - available on the OSX app store.
+- [akiyosi/gonvim](https://github.com/akiyosi/gonvim): A maintained fork of gonvim.
+- [etorth/libnvc](https://github.com/etorth/libnvc): An easy to embed GUI based on SDL.
+- [mvilim/neovim-pytc-example](https://github.com/mvilim/neovim-pytc-example): The first terminal UI for Neovim that isn't the default terminal UI!
+- [GNvim](https://github.com/vhakulinen/gnvim): "GUI for neovim, without any web bloat" written in Rust.
+- [Firenvim](https://github.com/glacambre/firenvim): A GUI for neovim full of web bloat because it embeds Neovim in Chrome, Firefox and other browsers.
+- [glrnvim](https://github.com/beeender/glrnvim): A terminal wrapper that launches neovim inside of alacritty.
+- [yatli/fvim](https://github.com/yatli/fvim): A beautiful UI written in f# - who knew functionnal languages could actually be used for writing software?
```

Comment:
hope that won't upset too many people :)

Reply:
I thought it was funny (especially since I prefer functional languages to imperative/object-oriented ones) but if you feel that's too much, I can remove it :).